.pre-commit-config.yaml

```diff
@@ -4,7 +4,6 @@ repos:
   rev: v1.19.0
   hooks:
     - id: terraform_fmt
-    - id: terraform_tflint
```

Comment:
why'd you remove this?

Reply:
it breaks commits if you don't have an aws credential